### ISSUE TITLE
Some system locales can make checks always pass

### ISSUE THREAD

**Issue by bumerankkk** [OpenMediaVault-Plugin-Developers/openmediavault-autoshutdown#47]
I got system locale=`cs_CZ.utf8`, that made for instance `top` command was showing load with decimal point as comma ',' instead of dot '.' (for instance **load average: 0,43, 0,14, 0,11**), as dot is expected in regular expression for extracting such information.
But I also noticed, that outside of UPHOURS, even if I was
- actively connected to Samba share from Win7 PC (thus in OMV WebUI > Diagnostics > Services >SMB/CIFS I saw my user as connected) 
- actively connected via SSH (in OMV WebUI > Diagnostics > Services >SSH I saw my user as connected) 
- running rclone (syncing tool between Linux and cloud storages such as Google drive, Dropbox, etc.) so the load was possibly higher then LOADAVERAGE but quite certainly traffic was higher than ULDLRATE

still I got my NAS regularly every 15 minutes (180s sleep * 5 cycles from my config) suspended (several times at row - really every 15minutes).

After changing my locale in config file to `en_US.utf8` like is here
```
# cat /etc/default/locale
LANG="en_US.UTF-8"
```
seems it works as expected.

Here is my complete config:
```
# cat /etc/autoshutdown.conf
ENABLE="true"
CYCLES=5
SLEEP=180
RANGE="2..98,100..200,202..254"
SHUTDOWNCOMMAND="systemctl suspend"
CHECKCLOCKACTIVE="true"
UPHOURS="17..22"
NSOCKETNUMBERS="21,22,80,139,443,445,3689,6991,8080,9091,49152"
ULDLCHECK="true"
ULDLRATE=50
LOADAVERAGECHECK="true"
LOADAVERAGE=16
HDDIOCHECK="true"
HDDIO_RATE=401
CHECK_SAMBA="true"
CHECK_CLI="true"
SYSLOG="true"
VERBOSE="true"
FAKE="false"
LOADPROCNAMES="smbd,nfsd,transmission-daemon,mt-daapd,forked-daapd,rclone"
```

**Comment by nrandon** [comment]
Should be addressed in 5.1.4

**Comment by nrandon** [comment]
this has been fixed, if a new problem relating to system localisation is found we should raze a new issue
